compliance-report/generate.sh

```diff
@@ -21,12 +21,17 @@ elif [ -n "$SECURITY_REPORT_URL" ]; then
     SECURITY_REPORT="See $SECURITY_REPORT_URL"
 fi
 
+THIRD_PARTY_DEPENDENCY_TRACKER="n/a"
+if [ -n "$THIRD_PARTY_DEPENDENCY_TOOL" ]; then
+    THIRD_PARTY_DEPENDENCY_TRACKER="$THIRD_PARTY_DEPENDENCY_TOOL"
+fi
+
```

Comment:
Nit: Couldn't this be replaced with a `default: Silk` line in the corresponding action.yml (in keeping with Steve's suggestion for BC)?

Reply:
Done